curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/basic-javascript/chaining-if-else-statements.portuguese.md

```diff
@@ -77,6 +77,6 @@ testSize(7);
 <section id='solution'>
 
 ```js
-// solution required
+  Solução necessária
```

Comment:
This should be kept as a comment so it doesn't throw errors when the user doesn't remove them.